examples/rails52/.proxyconfig/htpasswd

```diff
@@ -0,0 +1 @@
+pxuser:$apr1$wr8E9m1.$Jyz5NrKJf7I3x19a2NYgH0
```

Comment:
Is this a fake password?

Reply:
Yep! This is for the `pxuser` used for authenticated forward proxying that is [set in the docker-compose.yml](https://github.com/honeycombio/beeline-ruby/pull/141/files#diff-191eb201a4025ad4612d188098339ec4c44e8ca770a4fe8aae539ef731c5da06R14). It is only used there in this wee standalone environment and not a _secret_ secret.